portal-frontend/src/app/shared/owner-dialogs/parcel-owners/parcel-owners.component.ts

```diff
@@ -92,13 +93,17 @@ export class ParcelOwnersComponent {
       });
     }
     dialog.beforeClosed().subscribe((updatedUuid) => {
-      if (updatedUuid) {
+      if (updatedUuid?.ownerDeleted) {
+        this.onOwnersDeleted.emit(updatedUuid.deletedOwnerId);
+      }
+      else if (updatedUuid) {
         this.onOwnersUpdated.emit();
```

Comment:
onOwnersUpdated told the parent component to re-query the owners from the backend. Is there a reason this is not sufficient at updating the table?